### PR TITLE
Add npm configuration for publishing

### PR DESCRIPTION
**Pull request by bensampaio**
Added the file `Resources/package.json`.

If you run `npm publish` in the `Resources` folder, the package will be published only with the folders `js` and `public/js`. This way the JS is still part of the bundle but is also available through npm.

Making the 'Translator' available through npm allows JS developers to `require` / `import` it when using `webpack`, `jspm`, or other modules bundler. And also allows the separation of back-end and front-end dependencies.

### CODE REVIEW

**Comment by monteiro** [comment]
@bensampaio yes, you are right. I am not using webpack to import this module, but I should as well. 

Can you write a little section in https://github.com/willdurand/BazingaJsTranslationBundle/blob/master/Resources/doc/index.md to explain how can we import this via webpack for example?

File: Resources/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "bazinga-translator",
+  "version": "1.0.0",
```
Comment (monteiro) [inline Resources/package.json]: The version used should be the same as the BazingaJsBundle. That way we keep both on track, if in the future we want to update the javascript. Like reactjs does in all of their internal components.

File: Resources/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "email": "will+git@drnd.me"
+  },
+  "contributors": [
+    {
+      "name": "Bruno Sampaio",
+      "email": "bens.sampaio@gmail.com"
+    }
+  ],
+  "files": [
+    "js",
```
Comment (stof) [inline Resources/package.json]: you should include only ``js/translator.js``. The folder includes a bunch of other files which are needed only by the testsuite

**Comment by bensampaio** [comment]
I changed the version in the `package.json` and included only the `translator` files in it.

**Comment by monteiro** [comment]
@bensampaio now the only thing missing is to explain it in https://github.com/willdurand/BazingaJsTranslationBundle/blob/master/Resources/doc/index.md :)

**Comment by bensampaio** [comment]
@monteiro I just did that, what do you think?

File: Resources/doc/index.md [hunk 3]
```diff
@@ -44,16 +46,36 @@ Publish assets:
 
     php app/console assets:install --symlink web
 
+### Require via NPM
```
Comment (stof) [inline Resources/doc/index.md]: this is missing an explanation saying that it is optional (it is only needed when wanting to import the translator using modules).

and you should also document that it is very strongly recommended to keep the version in sync between the npm package and the bundle version being used.

Comment (bensampaio) [inline Resources/doc/index.md]: Just did that ;) thanks!

**Comment by monteiro** [comment]
@bensampaio Awesome! Thanks!

**Comment by bensampaio** [comment]
Thanks! Don't forget to run `npm publish` on the `Resources` folder so it becomes available on npm :)

**Comment by monteiro** [comment]
@bensampaio Thanks! I will do it tomorrow ;)

**Comment by monteiro** [comment]
@bensampaio already published

screen shot 2016-07-26 at 13 47 54

**Pull request merged**